sycl/test/abi/sycl_symbols_windows.dump

```diff
@@ -1,4894 +1,4894 @@
-?sycl_host_native_exp@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_exp@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_exp@__host_std@@YAMM@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_log10@__host_std@@YAMM@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_log2@__host_std@@YAMM@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_log@__host_std@@YAMM@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@0@Z
-?sycl_host_native_powr@__host_std@@YAMMM@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_recip@__host_std@@YAMM@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$00@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$01@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$02@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$03@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$07@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YA?AV?$vec@M$0BA@@_V1@sycl@@V234@@Z
-?sycl_host_native_rsqrt@__host_std@@YAMM@Z
-?sy
```

Comment:
Not sure why Github marks same lines as modified, may be because of modified EOL symbols from Win to UNIX or from UNIX to Win. Could you please check and if this is like that, and keep the old EOL symbols?

Reply:
Yes, I think I might have lost the EOL symbols due to the way I generated the file. I will try to regenerate it.